src/schema/notifications.ts

```diff
@@ -103,14 +103,19 @@ export class ZimbraNotifications {
 			items.forEach((i: any) => {
 				const item = normalizeContact(i);
 				const defaultFolderName = 'Contacts';
-				const folder: any = this.cache.readFragment({
-					id: `Folder:${item.folderId}`,
-					fragment: gql`
-					fragment folderName${item.folderId} on Folder {
-						name
-					}
+				let folder: any;
+				try {
+					folder = this.cache.readFragment({
+						id: `Folder:${item.folderId}`,
+						fragment: gql`
+						fragment ${generateFragmentName('folderName', item.folderId)} on Folder {
```

Comment:
where are we normalizing name when writing to cache? or the cache will have : in folder id?

Reply:
Yes folder id is coming with ":"